From: "David S. Miller" <davem@davemloft.net>
Date: Tue, 20 Apr 2010 18:49:45 -0700
Subject: [PATCH 120/207] tg3: Enable GRO by default.

commit cb903bf4ee2d6e53210e2174d363e10698112042 upstream.

This was merely an oversight when I added the *_gro_receive()
calls.

Signed-off-by: David S. Miller <davem@davemloft.net>
---
 drivers/net/tg3.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/net/tg3.c b/drivers/net/tg3.c
index bfbe11e..72bbdcb 100644
--- a/drivers/net/tg3.c
+++ b/drivers/net/tg3.c
@@ -12994,6 +12994,7 @@ static int __devinit tg3_get_invariants(struct tg3 *tp)
 		tp->dev->features |= NETIF_F_IP_CSUM | NETIF_F_SG;
 		if (tp->tg3_flags3 & TG3_FLG3_5755_PLUS)
 			tp->dev->features |= NETIF_F_IPV6_CSUM;
+		tp->dev->features |= NETIF_F_GRO;
 	}
 
 	/* Determine TSO capabilities */
-- 
1.7.5.4

